### DISCUSSION TITLE
ScrollView initial scroll before mount

### DISCUSSION THREAD

**Discussion by kdkavanagh** [Textualize/textual#4725] [Q&A]
Is there a mechanism to set the initial X/Y for a `ScrollView` before the widget is mounted?  Currently I'm using `obj.scroll_to()` after `await self.mount(obj)`, which works, but ends up flashing the un-scrolled view first before scrolling

Calling `scroll_to` before mounting seems to have no effect

**Comment by darrenburns** [comment]
No, I don't think there's a mechanism for that yet. I've hit the same issue before and agree it would be very useful.
